docs/core.adoc

```diff
@@ -0,0 +1,31 @@
+= AeroGear Services Core SDK
+
+The Core SDK provides helpers for parsing the configuration file and share common utilities between other SDK modules.
+
+It's main features are:
+
+* Parsing of the configuration file
+* Logging
+* Metrics gathering
+
+== Installation
+
+Use the Cordova CLI to add the Core module as a plugin to your project:
+
+----
+cordova plugin add @aerogearservices/Core
```

Comment:
Lowercase?

Reply:
`core-cordova` is not valid js identifier. While npm package needs to be named `core-cordova` namespace do not need to as we know that we are in the cordova ecosystem anyway.

Reply:
Awesome, thanks for that @wtrocki makes sense!

Reply:
@wtrocki But how do we install the plugin from NPM using `@aerogearservices/core` it that's already a package. If we don't want dashes I would go for `corecordova`